vite.config.ts

```diff
@@ -25,8 +25,8 @@ export default defineConfig({
     extensions: ['.mjs', '.js', '.ts', '.jsx', '.tsx', '.json', '.vue']
   },
   test: {
-    // simulate DOM with happy-dom
-    environment: 'happy-dom',
+    // simulate DOM with jsdom
+    environment: 'jsdom',
```

Comment:
The unit tests pass.

![image](https://github.com/bcgov/bcrs-shared-components/assets/10002889/7e0751ff-b4e5-4f28-81c5-c3518d2ba688)